### ISSUE TITLE
ui5-busyindicator: Loading Icon Not Announced by Screen Reader

### ISSUE THREAD

**Issue by GongRichard** [SAP/ui5-webcomponents#3591]
### **Bug Description**
When the busy indicator is showing, the screen reader won't read out busy text.

### **Expected Behavior**
When the busy indicator is showing, the screen reader should read out busy text.

### **Steps to Reproduce**
1. Open JAWS or NVDA
2. Go to 'https://sap.github.io/ui5-webcomponents/playground/components/BusyIndicator/'
3. Click 'Fetch List Data' button
4. JAWS or NVDA won't read out busy text

### **Context**
- UI5 Web Components version: 0.31.12
- OS/Platform: Windows
- Browser: Chrome
- Affected component: ui5-busyindicator

### **Log Output / Stack Trace / Screenshots**
![2021-08-04_17-19-08](https://user-images.githubusercontent.com/13098801/128162524-3cfea643-37aa-4b55-8109-6f605c382194.jpg)

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

### **Stakeholder Info** (if applicable)
- Organization: SAP SuccessFactors

**Comment by elenastoyanovaa** [comment]
Hello @SAP/ui5-webcomponents-topic-rd ,

When a busy indicator is added dinamically (busy state of an component/control), nothing is read out. Since the same is also reproducible for the UI5 Busy Indicator https://sapui5.hana.ondemand.com/#/entity/sap.ui.core.Control/sample/sap.ui.core.sample.ControlBusyIndicator perhaps a change in the specification and handling of busy controls should be discussed with ACC experts and handled. Not sure if this should be handled as an issue or a feature request. There is also a BLI in UI5 maybe related to this, please check the wiki of the busy indicator control and process further.

Kind Regards,
Elena

**Comment by gmkv** [comment]
Hi @GongRichard 

Can you let me know the steps (where you are exactly on the page/which element you have focused) that you took to reproduce this issue?

The ui5-busy-indicator will not announce that it is loading if it is contained in a different region of the page.
To users of screen readers, it doesn't make sense to announce that a busy indicator situatied in another region of the page is loading, while they are focused on a completely different region of the page.

Regards,
Georgi

**Comment by gmkv** [comment]
Related to Alexandar's comment on [#3186](https://github.com/SAP/ui5-webcomponents/issues/3186#issuecomment-849731105)

**Comment by GongRichard** [comment]
Hi @gmkv ,

Would you please give this issue more high priority as SF is close to the code complete date which is Sep 19th?

Thanks,
Richard

**Comment by gmkv** [comment]
Hi @GongRichard 

We discussed the SR announcement with our accessibility experts for the ui5-busy-indicator, and their recommendation is that it will not be desirable for the change in the "busy" state to be announced.
Consider the case where there are multiple busy indicators on a page that become busy at the same time. This would result in each busy indicator being announced immediately one after the other, which will be less helpful than useful to a user with a screen reader.

From this issue, we also discovered a bug which doesn't allow the busy-indicator to be focused - we will fix this.

For your application, our advice is to open your dialog with the initial focus set on the table (which internally has a busy indicator). After it loads, you will continue as usual, by setting the focus to be on the "Select All" checkbox as it is currently implemented.

I will let you know when we have an update for this.

Regards,
Georgi

**Comment by gmkv** [comment]
I will close this issue as consulation, because there is not a real bug with it. I have opened a seperate issue #3922 for us to handle the bug I described above.
